core/parameters/src/cost.rs

```diff
@@ -432,6 +432,21 @@ pub struct RuntimeFeesConfig {
     ///
     /// Changed to false with [NEP-536](https://github.com/near/NEPs/pull/536)
     pub refund_gas_price_changes: bool,
+
+    /// Relative cost for gas refunds as a ratio of the refunded amount.
+    ///
+    /// The actual penalty is
+    /// `max(gross_refund * gas_refund_penalty, min_gas_refund_penalty)`
+    ///
+    /// Added with [NEP-536](https://github.com/near/NEPs/pull/536)
+    pub gas_refund_penalty: Rational32,
```

Comment:
If this has to be a non-negative rational, consider `Rational<u32>`? It would probably also be a good idea to ensure that this penalty can't be `> 1`, but not sure how to enforce that well (maybe a plain unit test?)